ambari-web/app/models/host_component.js

```diff
@@ -21,6 +21,7 @@ var App = require('app');
 App.HostComponent = DS.Model.extend({
   workStatus: DS.attr('string'),
   passiveState: DS.attr('string'),
+  compId: DS.attr('string'),
```

Comment:
Please name this componentId and make it a number type

Reply:
I have used compId across the entire commit and should be fine as long as it is consistent. Also there is no particular need for this to be a number as this is an identifier and would not be used in any calculation. In the backend too this is a string. In future the id could not be a number but a combination of some string like a few other models.